.aios-core/cli/commands/config/index.js

```
@@ -0,0 +1,607 @@
+    }
+
+    // Check existing split files
+    const fwPath = path.join(root, CONFIG_FILES.framework);
+    const projPath = path.join(root, CONFIG_FILES.project);
+    const localPath = path.join(root, CONFIG_FILES.local);
+
+    if (!options.force) {
+      const exists = [fwPath, projPath, localPath].filter(p => fs.existsSync(p));
+      if (exists.length > 0) {
+        console.error('Split config files already exist:');
+        exists.forEach(p => console.error(`  ${p}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)